SPECS/pam_krb5/pam_krb5.spec

```diff
@@ -0,0 +1,72 @@
+Summary:        A Pluggable Authentication Module for Kerberos 5
+Name:           pam_krb5
+Version:        4.11
+Release:        2%{?dist}
+License:        MIT AND BSD
+Group:          System/Libraries
+URL:            https://github.com/rra/pam-krb5
+Source0:        %{url}/archive/refs/tags/upstream/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Patch0001:      0001-Drop-module-long-test.patch
+BuildRequires:  byacc
+BuildRequires:  flex
+BuildRequires:  gcc
+BuildRequires:  krb5-devel
+BuildRequires:  pam-devel
+BuildRequires:  perl(Test::Pod)
+# for testing
+BuildRequires:  perl(lib)
+Requires:       pam
+
+%description
+pam-krb5 is a Kerberos v5 PAM module for either MIT Kerberos or
+Heimdal.  It supports ticket refreshing by screen savers, configurable
+authorization handling, authentication of non-local accounts for
+network services, password changing, and password expiration, as well
+as all the standard expected PAM features.  It works correctly with
+OpenSSH, even with ChallengeResponseAuthentication and
+PrivilegeSeparation enabled, and supports extensive configuration
+either by PAM options or in krb5.conf or both.  PKINIT is supported
+with recent versions of both MIT Kerberos and Heimdal and FAST is
+supported with recent MIT Kerberos.
+
+%prep
+%setup -q -n pam-krb5-upstream-%{version}
+
+%autopatch -p1
+
+%build
+%configure --libdir=%{_libdir}
+%make_build
+
+%install
+%make_install
+
+# Make the paths jive to avoid conflicts on multilib systems.
+sed -ri -e 's|/lib(64)?/|/\$LIB/|g' %{buildroot}/%{_mandir}/man*/pam_krb5*.5*
```

Comment:
Shouldn't be necessary for Mariner, since we don't support multilib systems

Reply:
Fixed